helix-syntax/src/lib.rs

```diff
@@ -13,7 +14,8 @@ macro_rules! mk_extern {
 #[macro_export]
 macro_rules! mk_enum {
     ( $( $camel:ident ),* ) => {
-        #[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
+        #[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize, EnumString)]
+        #[strum(ascii_case_insensitive)]
```

Comment:
Drop this `strum` thing. You need to look up based on `language.scope`, not `language.language_id`